src/main.cpp

```diff
@@ -746,7 +713,12 @@ bool IsStandardTx(const CTransaction& tx, string& reason, const int nHeight)
 
     unsigned int nDataOut = 0;
     txnouttype whichType;
-    BOOST_FOREACH(const CTxOut& txout, tx.GetVout()) {
+    BOOST_FOREACH(const CTxOut& txout, txBase.GetVout()) {
+
+        // if the output comes from a backward transfer (when we are a certificate), skip this check
+        // but go on if the certificate txout is an ordinary one
+        if (txout.isFromBackwardTransfer)
```

Comment:
Why are we skipping this check? We are also skipping the IsDust() check below.

Reply:
We should not skip this check, it would be better to explicitly skip the replay-protection related check few lines below,as is it done for coinbase.

Reply:
Agreed, as you explained AFK I'll skip only the isTransactionTypeAllowedAtHeight check for bwt and run the rest of checks on all outputs